app/controllers/doctor_patients_controller.rb

```diff
@@ -0,0 +1,9 @@
+class DoctorPatientsController < ApplicationController
+  
+  def destroy
+    doctor = Doctor.find(params[:id])
+    DoctorPatient.find_by(doctor_id: params[:id], patient_id: params[:format]).destroy
```

Comment:
If you find yourself doing something like `params[:format]` then you are not passing enough data to the controller action.